docs/tutorials/multimodal/document/pdf_classification.ipynb

```diff
@@ -0,0 +1,234 @@
+    "\n",
+    "download_dir = './ag_automm_tutorial_pdf_classifier'\n",
+    "zip_file = \"https://automl-mm-bench.s3.amazonaws.com/doc_classification/pdf_docs_small.zip\"\n",
+    "load_zip.unzip(zip_file, unzip_dir=download_dir)\n",
+    "\n",
+    "dataset_path = os.path.join(download_dir, \"pdf_docs_small\")\n",
+    "pdf_docs = pd.read_csv(f\"{dataset_path}/data.csv\")\n",
+    "train_data = pdf_docs.sample(frac=0.8, random_state=200)\n",
+    "test_data = pdf_docs.drop(train_data.index)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's visualize one of the PDF documents. Here, we use the S3 URL of the PDF document and `IFrame` to show it in the tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import IFrame\n",
+    "IFrame(\"https://automl-mm-bench.s3.amazonaws.com/doc_classification/historical_1.pdf\", width=400, height=500)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As you can see, this document is an America's historical document in PDF format. \n",
+    "To make sure the tutorial can locate the documents correctly, we need to overwrite the document paths."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from autogluon.multimodal.utils.misc import path_expander\n",
+    "\n",
+    "DOC_PATH_COL = \"doc_path\"\n",
+    "\n",
+    "train_data[DOC_PATH_COL] = train_data[DOC_PATH_COL].apply(lambda ele: path_expander(ele, base_folder=download_dir))\n",
+    "test_data[DOC_PATH_COL] = test_data[DOC_PATH_COL].apply(lambda ele: path_expander(ele, base_folder=download_dir))\n",
+    "print(test_data.head())"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a PDF Document Classifier\n",
+    "\n",
+    "You can create a PDFs classifier easily with `MultiModalPredictor`. \n",
+    "All you need to do is to create a predictor and fit it with the above training dataset. \n",
+    "AutoMM will handle all the details, like (1) detecting if it is PDF format datasets; (2) processing PDFs like converting it into a format that our model can recognize; (3) detecting and recognizing the text in PDF documents; etc., without your notice. \n",
+    "\n",
+    "Here, label is the name of the column that contains the target variable to predict, e.g., it is “label” in our example. \n",
+    "We set the training time limit to 120 seconds for demonstration purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from autogluon.multimodal import MultiModalPredictor\n",
+    "\n",
+    "predictor = MultiModalPredictor(label=\"label\", verbosity=5)\n",
```

Comment:
Using `verbosity=5` outputs some debugging info, which should be avoided in tutorials.